modin/core/io/text/text_file_dispatcher.py

```diff
@@ -641,6 +649,24 @@ def check_parameters_support(
         if read_kwargs["chunksize"] is not None:
             return False
 
+        skiprows_unsupported = (
+            is_list_like(skiprows_md) and skiprows_md[0] < header_size
+        ) or (
+            callable(skiprows)
+            and any(
+                map(
+                    bool,
+                    cls._get_skip_mask(pandas.RangeIndex(header_size), skiprows),
+                )
+            )
+        )
```

Comment:
This part of the condition is hard to read. You can, for example, save the result of `any(...)` function to a variable.

Reply:
Ok, code fragment was rewritten.